core/src/main/java/hudson/util/Iterators.java

```diff
@@ -403,4 +405,20 @@ public void remove() {
     public interface CountingPredicate<T> {
         boolean apply(int index, T input);
     }
+
+    /**
+     * Similar to {@link com.google.common.collect.Iterators#skip} except not {@link Beta}.
+     * @param iterator some iterator
+     * @param count a nonnegative count
+     * @since FIXME
```

Comment:
Or could be `@Restricted(NoExternalUse.class)`, though I saw no harm in making it available.

Reply:
IMHO better `@Restricted(NoExternalUse.class)` to reduce API surface

Reply:
+1

Reply:
Will do.